src/plone/restapi/tests/http-examples/translations_expand_get.req

```diff
@@ -0,0 +1,8 @@
+GET /plone/en/test-document?expand=@translations HTTP/1.1
```

Comment:
The request should read '/plone/en/test-document?expand=translations' without the '@' sign.
The request is a GET, so no body here.